corehq/apps/locations/models.py

```diff
@@ -374,6 +398,15 @@ def by_location_id(cls, location_id):
         except cls.DoesNotExist:
             return None
 
+    def linked_supply_point(self):
+        from corehq.apps.commtrack.models import SupplyPointCase
+        if not self.supply_point_id:
+            return None
+        try:
+            return SupplyPointCase.get(self.supply_point_id)
+        except:
```

Comment:
This definitely doesn't need to be a bare `except`, and I would fully support catching `CaseNotFound` instead.  I'm actually surprised that I did this, I wonder if I didn't know the exception at the moment and just forgot to go back and add it in.